File: Beginner/Exer. 1006/main.c
#include <stdio.h>
#define weightA 2.0
#define weightB 3.0
#define weightC 5.0

int main(int argc, char *argv[]) {
    double A, B, C;
    
    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);
    
    printf("MEDIA = %.1lf\n", ((A * weightA + B * weightB + C * weightC)) / (weightA + weightB + weightC));
    return 0;
}

File: Beginner/Exer. 1006/main.java
import java.util.Scanner;

class Main {
    public static final double weightA = 2.0;
    public static final double weightB = 3.0;
    public static final double weightC = 5.0;

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double A, B, C;
        A = scanner.nextDouble();
        B = scanner.nextDouble();
        C = scanner.nextDouble();
        scanner.close();

        System.out.printf("MEDIA = %.1f\n", ((A * weightA + B * weightB + C * weightC)) / (weightA + weightB + weightC));
    }
}